### ISSUE TITLE
Jquery and bootstrap-jquery ui no longer compatible 

### ISSUE THREAD

**Issue by Kaik** [zikula/core#3952]
| Q                 | A
| ----------------- | ---
| Zikula Version    | 2.x.x
| PHP Version       | 5.x

### Expected behavior
Working user search in routes module add custom route - it used to work before but for some reason, it stopped it might be my setup - it is kind of broken anyway... looks like bootstrap jquery ui (older) is loading after newer jquery-ui called by the live search... most probably priority will "solve" this issue...

### Actual behavior
```
jquery.js:10020 Uncaught TypeError: elem.getClientRects is not a function
    at jQuery.fn.init.offset (jquery.js:10020)
    at Object.getWithinInfo (bootstrap-jqueryui.min.js:5)
    at jQuery.fn.init.$.fn.position (bootstrap-jqueryui.min.js:5)
    at t.(anonymous function).(anonymous function)._suggest (http://localhost:81/web/jquery-ui/jquery-ui.min.js:8:5948)
    at t.(anonymous function).(anonymous function)._suggest (http://localhost:81/web/jquery-ui/jquery-ui.min.js:6:4499)
    at t.(anonymous function).(anonymous function).__response (http://localhost:81/web/jquery-ui/jquery-ui.min.js:8:5225)
    at t.(anonymous function).(anonymous function).__response (http://localhost:81/web/jquery-ui/jquery-ui.min.js:6:4499)
    at t.(anonymous function).(anonymous function).n (http://localhost:81/web/jquery-ui/jquery-ui.min.js:6:4384)
    at t.(anonymous function).(anonymous function).__response (http://localhost:81/web/jquery-ui/jquery-ui.min.js:8:7655)
    at t.(anonymous function).(anonymous function).__response (http://localhost:81/web/jquery-ui/jquery-ui.min.js:6:4499)
offset @ jquery.js:10020
getWithinInfo @ bootstrap-jqueryui.min.js:5
$.fn.position @ bootstrap-jqueryui.min.js:5
_suggest @ jquery-ui.min.js:8
(anonymous) @ jquery-ui.min.js:6
__response @ jquery-ui.min.js:8
(anonymous) @ jquery-ui.min.js:6
n @ jquery-ui.min.js:6
__response @ jquery-ui.min.js:8
(anonymous) @ jquery-ui.min.js:6
(anonymous) @ jquery-ui.min.js:8
proxy @ jquery.js:10268
(anonymous) @ Zikula.Users.LiveSearch.js:29
fire @ jquery.js:3268
fireWith @ jquery.js:3398
done @ jquery.js:9305
(anonymous) @ jquery.js:9548
load (async)
send @ jquery.js:9567
ajax @ jquery.js:9206
jQuery.(anonymous function) @ jquery.js:9355
getJSON @ jquery.js:9336
source @ Zikula.Users.LiveSearch.js:28
_search @ jquery-ui.min.js:8
(anonymous) @ jquery-ui.min.js:6
search @ jquery-ui.min.js:8
(anonymous) @ jquery-ui.min.js:6
(anonymous) @ jquery-ui.min.js:8
i @ jquery-ui.min.js:6
setTimeout (async)
_delay @ jquery-ui.min.js:6
_searchTimeout @ jquery-ui.min.js:8
(anonymous) @ jquery-ui.min.js:6
input @ jquery-ui.min.js:8
r @ jquery-ui.min.js:6
dispatch @ jquery.js:5183
elemData.handle @ jquery.js:4991
```
### Steps to reproduce
go to routes module add custom route there is a user search at the bottom try use this search...

### some ideas
I have a few issues with this setup. Everything starts with #3947 both users module and profile module are designed to return HTML which is bad...
Fix I suggest is to add 'raw' option - default set to false so BC is covered. When set to true only avatar url is returned. Use this with raw = true in live search controller. 

Rewrite whole live search to vanilla js or just Jquery and place everything in one place now form widget is in FormExtensionsBundle, css and js in UsersModule I'm not sure this is how it should be done.

**Comment by Guite** [comment]
This will be solved by #3931

**Comment by Guite** [comment]
Fixed.
